src/less/src/common.less

```diff
@@ -79,6 +79,8 @@ input:not([type=button]):not([type=submit]).w2ui-input, textarea.w2ui-input {
     background-color: @input-background-color;
     line-height: normal;
     &:focus {
+        outline-style: solid;
+        outline-width: 1px;
```

Comment:
sorry, I used firefox 41.0.2, without the two attributes, the focus field no any outline border appeared, I did not test it with other browser.